internal/core/arg_specs.go

```diff
@@ -62,6 +71,13 @@ type ArgSpec struct {
 
 	// ValidateFunc validates an argument.
 	ValidateFunc ArgSpecValidateFunc
+
+	// Positional defines whether the argument is a positional parameter.
+	Positional bool
```

Comment:
We need to check that we only have one positional argument in `ArgSpecs`. It could be useful for custom commands.